project/src/main/data/player-save-upgrader.gd

```diff
@@ -319,6 +320,28 @@ func new_save_item_upgrader() -> SaveItemUpgrader:
 	return upgrader
 
 
+func _post_upgrade_59c3(new_save_items: Array) -> Array:
```

Comment:
Add some comments describing what this code does. It's pretty dense and hard to understand if you didn't just work on it.

Reply:
Added comments, `locate the 'difficulty' save item`...`ensure the 'difficulty' save item`...